gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,126 @@ angular.module('syncthing.core')
             }
         };
 
+        $scope.deviceStatusIcon = function(cfg) {
+            switch ($scope.deviceStatus(cfg)) {
+                case 'insync':
+                    return 'fa-check';
+                case 'unused-insync':
+                    return 'fa-unlink';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'paused':
+                    return 'fa-pause';
+                case 'unused-paused':
+                    return 'fa-unlink';
+                case 'disconnected':
+                    return 'fa-power-off';
+                case 'disconnected-inactive':
+                    return 'fa-power-off';
+                case 'unused-disconnected':
+                    return 'fa-unlink';
+            }
+        };
+
+        $scope.deviceStatusText = function(device) {
+            switch ($scope.deviceStatus(device)) {
+                case 'insync':
+                    return $translate.instant('Up to Date');
+                case 'unused-insync':
+                    return $translate.instant('Connected (Unused)');
+                case 'syncing':
+                    return $translate.instant('Syncing') + ' (' + percentFilter($scope.completion[device.deviceID]._total) + ', ' + binaryFilter($scope.completion[device.deviceID]._needBytes) + 'B)';
+                case 'paused':
+                    return $translate.instant('Paused');
+                case 'unused-paused':
+                    return $translate.instant('Paused (Unused)');
+                case 'disconnected':
+                    return $translate.instant('Disconnected');
+                case 'disconnected-inactive':
+                    return $translate.instant('Disconnected (Inactive)');
+                case 'unused-disconnected':
+                    return $translate.instant('Disconnected (Unused)');
+            }
+        };
+
+        $scope.folderStatusIcon = function(cfg) {
+            switch ($scope.folderStatus(cfg)) {
+                case 'paused':
+                    return 'fa-pause';
+                case 'unknown':
+                    return 'fa-question-circle';
+                case 'unshared':
+                    return 'fa-unlink';
+                case 'scan-waiting':
+                    return 'fa-hourglass-half';
+                case 'cleaning':
+                    return 'fa-recycle';
+                case 'clean-waiting':
+                    return 'fa-hourglass-half';
+                case 'stopped':
+                    return 'fa-stop';
+                case 'scanning':
+                    return 'fa-search';
+                case 'idle':
+                    return 'fa-check';
+                case 'localadditions':
+                    return 'fa-check';
+                case 'sync-waiting':
+                    return 'fa-hourglass-half';
+                case 'sync-preparing':
+                    return 'fa-hourglass-half';
```

Comment:
```suggestion
                case 'sync-waiting':
                case 'sync-preparing':
                    return 'fa-hourglass-half';
```

And similar for all the other repeated values.